### PR TITLE
Change markdown code block language to support c# lang following lib `prism-react-renderer`

### PR DESCRIPTION
**Pull request by mildronize**
Replace markdown code block from `c#` to `cpp`, following the lib variables: https://github.com/FormidableLabs/prism-react-renderer/blob/master/src/vendor/prism/includeLangs.js#L11

```js
module.exports = {
...
  cpp: true,
...
};
```

Please review my PR.

### CODE REVIEW

File: content/programming-cookbook/logging/NLog.md [hunk 1]
```diff
@@ -50,7 +50,7 @@ install-package NLog
 
 - Set up the main class.
 
-```c#
+```cpp
```
Comment (aaronamm) [inline content/programming-cookbook/logging/NLog.md]: Does it support C#? Can we make C# work?

**Comment by mildronize** [comment]
add more css and prism lib to support `c#` lang

for markdown notation support only C# - `csharp`, `cs`, `dotnet`,

No support for `c#`

Ref:
https://prismjs.com/#supported-languages
https://github.com/FormidableLabs/prism-react-renderer#faq

File: package.json [hunk 2]
```diff
@@ -110,7 +110,8 @@
     "eslint-plugin-react": "7.21.4",
     "gatsby-plugin-remove-trailing-slashes": "2.3.11",
     "prettier": "2.0.5",
-    "prism-react-renderer": "1.1.1"
+    "prism-react-renderer": "1.1.1",
+    "prismjs": "^1.24.1"
```
Comment (aaronamm) [inline package.json:114]: 👍

File: src/components/MdxComponents/codeBlock.js [hunk 3]
```diff
@@ -1,9 +1,15 @@
 import * as React from 'react';
 import Highlight, { defaultProps } from 'prism-react-renderer';
+import Prism from "prism-react-renderer/prism";
 import theme from 'prism-react-renderer/themes/dracula';
 import Loadable from 'react-loadable';
 import LoadingProvider from './loading';
 
+/** Check if load Prism from browser or server */
```
Comment (aaronamm) [inline src/components/MdxComponents/codeBlock.js:8]: Nice comment, thanks.

File: src/components/MdxComponents/codeBlock.js [hunk 4]
```diff
@@ -1,9 +1,15 @@
...
+(typeof global !== "undefined" ? global : window).Prism = Prism;
+
+require("prismjs/components/prism-csharp");
```
Comment (aaronamm) [inline src/components/MdxComponents/codeBlock.js:11]: 👍 👍 👍 👍 👍

**Review by aaronamm — COMMENTED** [review]
@mildronize This is very useful PR. Thank you so much.

**Review by aaronamm — APPROVED** [review]

**Pull request merged**